spec.bs

```diff
@@ -746,9 +746,51 @@ A <dfn export for=fencedframetype>fenced frame reporter</dfn> is a [=struct=] wi
 
   1. Let |destination url| be |destination map|[|eventType|].
 
-  1. Send a beacon with |eventData| to |destination url|.
+  1. Let |request| be a new [=request=] with the following:
```

Comment:
Is there supposed to be a noun after "following"?

Reply:
Meh, idk. Requests aren't formal structs, so I don't want to say "items". I guess I could put "properties".